src/module/system/settings/variant-rules.ts

```diff
@@ -58,17 +58,16 @@ const SETTINGS: Record<string, SettingRegistration> = {
     proficiencyVariant: {
         name: "PF2E.SETTINGS.Variant.Proficiency.Name",
         hint: "PF2E.SETTINGS.Variant.Proficiency.Hint",
-        default: "ProficiencyWithLevel",
-        type: String,
-        choices: {
-            ProficiencyWithLevel: "PF2E.SETTINGS.Variant.Proficiency.Choices.ProficiencyWithLevel",
-            ProficiencyWithoutLevel: "PF2E.SETTINGS.Variant.Proficiency.Choices.ProficiencyWithoutLevel",
+        default: false,
+        type: Boolean,
+        onChange: () => {
+            resetActors(game.actors.filter((a) => a.type === "character"));
         },
     },
     proficiencyUntrainedModifier: {
         name: "PF2E.SETTINGS.Variant.UntrainedModifier.Name",
         hint: "PF2E.SETTINGS.Variant.UntrainedModifier.Hint",
-        default: 0,
+        default: -2,
```

Comment:
Is this correct? Untrained just adds level + 0. Why -2?

Reply:
https://2e.aonprd.com/Rules.aspx?ID=1370

> The initial implementation is fairly straightforward: the proficiency bonus just becomes +2 for trained, +4 for expert, +6 for master, and +8 for legendary. We recommend giving an untrained character a –2 proficiency modifier instead of a +0 proficiency bonus.

Reply:
Interesting. And these fields are only used in pwol. Sure!